WordPress/Classes/ViewRelated/Reader/ReaderReblogAction.swift

```diff
@@ -1,5 +1,9 @@
 /// Encapsulates a command to reblog a post
 class ReaderReblogAction {
+    // tells if the origin is the reader list or detail, for analytics purposes
+    enum OriginType {
+        case list, detail
```

Comment:
This is just a style preference, but we tend to declare the cases one per line, like so:
```
case list
case detail
```

Reply:
👍